webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
+const packagejson = require('./package.json');
+
+const dashLibraryName = packagejson.name.replace(/-/g, '_');
+
+module.exports = {
+    entry: {main: './src/index.js'},
+    output: {
+        path: path.resolve(__dirname, dashLibraryName),
+        filename: 'bundle.js',
+        library: dashLibraryName,
+        libraryTarget: 'window'
+    },
+    externals: {
+        react: 'React',
+        'react-dom': 'ReactDOM',
+        'plotly.js': 'Plotly'
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: 'babel-loader'
+                }
+            },
+            {
+                test: /\.css$/,
+                use: [
+                    {
+                        loader: 'style-loader'
+                    },
+                    {
+                        loader: 'css-loader'
+                    }
+                ]
+            }
+        ]
+    }
+};
```

Comment:
What's the reason for this new `webpack.config.js` file? The webpack config stuff for the components is in `./config/webpack/`, won't it be confusing to have 2 config files for webpack?

Reply:
The old configs were for webpack 3, this is for webpack 4.